lib/rules/no-useless-escape.js

```diff
@@ -9,6 +9,19 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
+/**
+* Returns the union of two sets.
+* @param {Set} setA The first set
+* @param {Set} setB The second set
+* @returns {Set} The union of the two sets
+*/
+function union(setA, setB) {
+    return new Set(function *() {
```

Comment:
We have the [`generator-star-spacing`](http://eslint.org/docs/rules/generator-star-spacing) rule configured to enforce this spacing.